stdlib/public/SwiftShims/HeapObject.h

```diff
@@ -182,6 +190,14 @@ static_assert(alignof(HeapObject) == alignof(void*),
   (unsigned) SWIFT_ABI_DEFAULT_OBJC_NUM_RESERVED_LOW_BITS
 #endif
 
+/// BridgeObject masks
+#ifdef __x86_64__
+// x86-64 uses a different #define from the others
+#define _swift_BridgeObject_TaggedPointerBits SWIFT_ABI_X86_64_BRIDGEOBJECT_TAG
```

Comment:
This part doesn't really make sense either. BridgeObject just always uses the high bit as a tag, regardless of what the underlying tagged pointer mechanism uses.

Reply:
Agreed. I think I was too caught up in how it was previously defined. I've redone it to be totally separate from anything ObjC.